pkg/httphelper/httphelper.go

```diff
@@ -6,14 +6,69 @@ import (
 	"net/http"
 )
 
-func Error(w http.ResponseWriter, err error) {
+type ErrorCode int
+
+const (
+	NotFoundError ErrorCode = iota
+	ObjectExistsError
+	SyntaxError
+	ValidationError
+	ServerError
+)
+
+var errorStrings = map[ErrorCode]string{
+	NotFoundError:     "not_found",
```

Comment:
I agree, distinguishing between "object not found" and "route not found" is important.